.ko.yaml

```diff
@@ -2,3 +2,11 @@ baseImageOverrides:
   github.com/tektoncd/pipeline/cmd/creds-init: gcr.io/tekton-nightly/github.com/tektoncd/pipeline/build-base:latest
   github.com/tektoncd/pipeline/cmd/git-init: gcr.io/tekton-nightly/github.com/tektoncd/pipeline/build-base:latest
   github.com/tektoncd/pipeline/cmd/entrypoint: busybox  # image must have `cp` in $PATH
+baseBuildOverrides:
+  github.com/tektoncd/pipeline/cmd/controller:
+    env:
+    - name: CGO_ENABLED
```

Comment:
Why is this necessary? Can you add a comment or link to more info?

Reply:
https://github.com/tektoncd/pipeline/commit/f85c53f047095e09d2017552877cb3fbac885b1e
This got added in that commit. I'm just keeping "in sync" those two for the time being :wink:

Reply:
Maybe a comment to explain makes sense? Looking at https://github.com/tektoncd/pipeline/commit/f85c53f047095e09d2017552877cb3fbac885b1e I don't quite understand why this was added in that case either